holidays/locale/th/LC_MESSAGES/CN.po

```diff
@@ -0,0 +1,61 @@
+# Chinese holidays th translation.
```

Comment:
It seems we use term "localization" everywhere else

Reply:
Good catch, fixed.